keyboards/diverge3/keymaps/workman/config.h

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2017 IslandMan93
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the boiler place with just `#pragma once`? 

Eg, the file should look like: 
```c
#pragma once

// place overrides here
#define PERMISSIVE_HOLD
#define TAPPING_TERM 150
```

And the include is not necessary, as they keyboard's config. includes it already

Reply:
Done.